docs/content/latest/explore/ysql-language-features/sql-feature-support.md

```diff
@@ -76,7 +76,7 @@ Altering tables | ✓ | [`ALTER TABLE`  documentation](https://docs.yugabyte.com
 
  Component | Supported |  Documentation|
 -----------|-----------|---------
- Indexes | ✓ | [Indexes documentation](https://docs.yugabyte.com/latest/explore/indexes-constraints/overview/)|
+ Indexes | ✓ | [Indexes documentation](https://docs.yugabyte.com/latest/explore/indexes-constraints/)|
```

Comment:
Ugh why aren't these all relative links??? (Answer: this was from a PR that didn't get adequately reviewed... I'll catch them all sometime.)

Reply:
I was also wondering about this, but didn't want to just change it.